content/templates/forlenget-saksbehandlingstid/template_en.hbs

```diff
@@ -17,7 +17,17 @@ We expect to be able to process {{felles.søkerNavn}}'s application within {{beh
 {{/eq}}{{else}}{{#eq variantType "FORLENGET"}}
 We expect to be able to process your application within {{behandlingsfristUker}} weeks.
 {{else}}{{#eq variantType "MEDLEM"}}
-We have to wait until the {{#eq antallBarn 1}}child is{{else}}children are{{/eq}} born before we can process your application, because we have no information that you are working in Norway.
+To be entitled to a lump sum grant, you must be a member of the Norwegian National Insurance Scheme when the child is born. We know little about your situation, and need more information to be able to assess whether you are entitled to lump sum grant.
```

Comment:
Må man ha "the lump sum grant" ? 

Ellers kom chat gpt med følgende forslag:
"We have limited information about your situation and need more details to assess whether you qualify for the lump sum grant."